user/manager.go

```diff
@@ -427,6 +431,15 @@ const (
 	migrate4To5UpdateQueries = `
 		UPDATE user_access SET topic = REPLACE(topic, '_', '\_');
 	`
+
+	// 5 -> 6
+	migrate5To6UpdateQueries = `
+		ALTER TABLE user ADD COLUMN provisioned INT NOT NULL DEFAULT (0);
+		ALTER TABLE user ALTER COLUMN provisioned DROP DEFAULT;
```

Comment:
I haven't seen this before, where a column is added with a default and then the default is immediately removed. Is this just a sqlite quirk?